components/infobox/wikis/apexlegends/infobox_person_player_custom.lua

```diff
@@ -130,6 +131,7 @@ end
 function CustomPlayer:adjustLPDB(lpdbData)
 	lpdbData.extradata.role = Variables.varDefault('role')
 	lpdbData.extradata.role2 = Variables.varDefault('role2')
+	lpdbData.extradata.input = Variables.varDefault('input')
```

Comment:
I don't see this defined anywhere, is it tested to make sure this work?

Reply:
Currently testing on /dev